code/datums/components/mood.dm

```diff
@@ -77,12 +77,20 @@
 			msg += "<span class='nicegreen'>I love life!</span>\n"
 
 	msg += "<span class='notice'>Moodlets:\n</span>"//All moodlets
-	if(mood_events.len)
-		for(var/i in mood_events)
-			var/datum/mood_event/event = mood_events[i]
-			msg += "[event.description]\n"
-	else
-		msg += "<span class='nicegreen'>I don't have much of a reaction to anything right now.<span>\n"
+	var/mood_msg = null
+	var/tought_msg = null
```

Comment:
```suggestion
	var/mood_msg = ""
	var/tought_msg = ""
```
These should start as strings since you are doing += on them, += on null is dodgy.